src/SmartComponents/CompletedTaskDetails/CompletedTaskDetails.js

```diff
@@ -161,6 +167,22 @@ const CompletedTaskDetails = () => {
     [completedTaskJobs]
   );
 
+  const hasReportJson = useMemo(
+    () =>
+      !tableLoading &&
+      completedTaskJobs.some(
+        (job) => job.results?.report_json?.entries !== undefined
+      ),
+    [completedTaskJobs, tableLoading]
+  );
+
+  const hasPlainReport = useMemo(
+    () =>
+      !tableLoading &&
+      completedTaskJobs.some((job) => job.results?.report !== undefined),
+    [completedTaskJobs, tableLoading]
+  );
+
```

Comment:
Not yet sure I want to keep this useMemo here... but on the other hand don't want to iterate over completedTaskJobs with each re-render. Any ideas?